shared/fs/folder/offline.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import * as Kb from '../../common-adapters'
+import React from 'react'
+import * as Styles from '../../styles/index'
+
+// TODO: what is the correct thing to do when don't have any props
+class OfflineFolder extends React.PureComponent<any> {
+  render() {
+    return (
+      <Kb.Box2 direction="vertical" fullHeight={true} fullWidth={true}>
```

Comment:
I know `SelfReset` does that, but I think both this and that should just have a `flex: 1` style instead of having `fullHeight={true}`. The reason is there are other stuff in the same outer container with this, and having `fullHeight={true}` here makes it `width: '100%'` which would squeeze other stuff. (`WithContent` above is a an example.)

I guess an alternative is to move [this](https://github.com/keybase/client/pull/16848/files#diff-6df9d6db4b7ccf7cbc83bcd2ba7ee713R65) into `WithContent`, then it'd be the only children of its parent and `fullHeight` would be fine.

Reply:
Also, we need to add the top-bar here, for the the Sync toggle. It's not there yet (in PR https://github.com/keybase/client/pull/16859/files#diff-493e2e04422d21c2ab95f3ec21f9e4dfR24). We'll also need to have a way to disable sort in this case.